src/config/graphical-session/graphics-drivers/optimus.md

```diff
@@ -51,7 +51,7 @@ Offloading Graphics Display with RandR 1.4
 - allows to switch to the NVIDIA GPU on a per-application basis
 - `nouveau` is a reverse-engineered driver and offers poor performance
 
-> Note: different methods are mutually exclusive.
+Note: different methods are mutually exclusive.
```

Comment:
not a note?

Reply:
Yeah, this is an example of text that should not be a note, but simply standard text. So it should just be something like "The above methods are mutually exclusive". But i think it would be better to just remove this line entirely, and change the line "A summary of the methods supported by Void:" to something like "The following mutually-exclusive methods are supported by Void:".

Reply:
I'm going to change this to "The above methods are mutually exclusive". I'll take a closer look at this file later and change it as you described